fetch/api/redirect/redirect-schemes.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Fetch: handling different schemes in redirects</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id=log></div>
+<script>
+  // All non-HTTP(S) schemes cannot survive redirects
+  var url = "../resources/redirect.py?location=",
+      tests = [
+    fetch(url + "mailto:a@a.com"),
+    fetch(url + "data:,HI"),
+    fetch(url + "facetime:a@a.org"),
+    fetch(url + "about:blank"),
+    fetch(url + "about:unicorn"),
+    fetch(url + "blob:djfksfjs")
+  ]
+  ;tests.forEach(function(f) {
+  	promise_test(function(t) {
+      return promise_rejects(t, new TypeError(), f)
```

Comment:
Sorry, removed the tabs. Leading with ";" where needed is sort of my preferred style, but I guess I can change it if you want.